extensions/amp-dynamic-css-classes/0.1/amp-dynamic-css-classes.js

```diff
@@ -48,6 +66,30 @@ export function referrers_(referrer) {
 }
 
 /**
+ * Normalizes certain referrers across devices.
+ * @param {!Window} win
+ * @returns {!Array<string>}
+ */
+function normalizedReferrers(win) {
+  const referrer = referrerDomain(win);
+  let referrers = referrers_(referrer);
+
+  // Normalize t.co names to twitter.com
+  if (referrer === 't.co') {
+    referrers = referrers.concat(referrers_('twitter.com'));
```

Comment:
Yup, this will include the both of them. Per my talk with @cramforce yesterday:

> Should we normalize `t.co` links to `twitter.com`?

> That would be cool, yeah
Because I want them to get rid of t.co
And if they did we likely would put the opposite hack on our side

So it seems to me we should allow people to use `t.co` if they're aware of twitter's redirecter, or `twitter.com` if they're not.